pkg/scheduler/cache/cache.go

```diff
@@ -283,6 +283,21 @@ func (cache *schedulerCache) processPodResourcesResizeRequest(newPod *v1.Pod) er
 		resizeResourcesPolicy = api.PodResourcesResizePolicy(newPod.ObjectMeta.Annotations[api.AnnotationResizeResourcesPolicy])
 	}
 
+	// If pod resources resize status has been set, clear out action and backup annotations.
+	for _, podCondition := range newPod.Status.Conditions {
+		if podCondition.Type == v1.PodResourcesResizeStatus {
+			actionVer, _ := newPod.ObjectMeta.Annotations[api.AnnotationResizeResourcesActionVer]
+			if podCondition.Message == actionVer {
+				// TODO: If ResizeStatus shows failure, restore previous values
```

Comment:
restore previous values in pod will cause controller to try to add the annotation again.

Reply:
perhaps we should do the rollback in the controller